src/schema.rs

```diff
@@ -403,51 +760,120 @@ pub struct Meter {
 }
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+/// Represents the live ruleset (to be [flushed][NfCmd::Flush]).
 pub struct Ruleset {}
 
+/// Default ruleset.
+impl Default for Ruleset {
+    fn default() -> Self {
+        Ruleset {}
+    }
+}
+
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+/// Library information in output.
+///
+/// In output, the first object in an nftables array is a special one containing library information.
 pub struct MetainfoObject {
     #[serde(skip_serializing_if = "Option::is_none")]
+    /// The value of version property is equal to the package version as printed by `nft -v`.
     pub version: Option<String>,
+    /// The value of release_name property is equal to the release name as printed by `nft -v`.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub release_name: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    /// The JSON schema version.
```

Comment:
I think JSON Schema can be treated as a proper name.

```suggestion
    /// The JSON Schema version.
```